### PR TITLE
Added `cargo_config` attribute to `cargo_bootstrap_repository`.

### PR DESCRIPTION
**Pull request by UebelAndre**
This can be useful for workspaces which have a `config.toml` for the workspace (used by `crate_universe`) and want to share the same configuration with any bootstrap repositories.

### CODE REVIEW

File: cargo/private/cargo_bootstrap.bzl [hunk 1]
```diff
@@ -237,13 +250,17 @@ cargo_bootstrap_repository = repository_rule(
             ],
             default = "release",
         ),
+        "cargo_config": attr.label(
+            doc = "The path of the Cargo configuration (`Config.toml`) file.",
+            allow_single_file = ["Cargo.toml"],
```
Comment (illicitonion) [inline cargo/private/cargo_bootstrap.bzl:255]: I don't think these files are typically called Cargo.toml?

**Review by illicitonion — APPROVED** [review]
Makes sense, but one question

File: cargo/private/cargo_bootstrap.bzl (hunk 1 above)
Comment (UebelAndre) [reply cargo/private/cargo_bootstrap.bzl:255]: Oh no!

Comment (UebelAndre) [reply cargo/private/cargo_bootstrap.bzl]: Fixed!

**Pull request merged**